quickwit-common/src/lib.rs

```diff
@@ -24,6 +24,9 @@ use std::net::{SocketAddr, ToSocketAddrs};
 use once_cell::sync::Lazy;
 use regex::Regex;
 
+/// Filenames used for hotcache files.
+pub const HOTCACHE_FILENAME: &str = "hotcache";
```

Comment:
Why in common?
It would make more sense in quickwit-storage?

Reply:
because it's also used in directories, the hotcache is a directory and we already have a directories -> storage dependency